File: specification/network/resource-manager/Microsoft.Network/stable/2022-07-01/examples-java/NetworkProfileGetConfigOnly.java
import com.azure.core.util.Context;

/** Samples for NetworkProfiles GetByResourceGroup. */
public final class Main {
    /*
     * x-ms-original-file: specification/network/resource-manager/Microsoft.Network/stable/2022-07-01/examples/NetworkProfileGetConfigOnly.json
     */
    /**
     * Sample code: Get network profile.
     *
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void getNetworkProfile(com.azure.resourcemanager.AzureResourceManager azure) {
        azure
            .networks()
            .manager()
            .serviceClient()
            .getNetworkProfiles()
            .getByResourceGroupWithResponse("rg1", "networkProfile1", null, Context.NONE);
    }
}

File: specification/network/resource-manager/Microsoft.Network/stable/2024-03-01/examples-js/NetworkProfileGetConfigOnly.js
const { NetworkManagementClient } = require("@azure/arm-network");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Gets the specified network profile in a specified resource group.
 *
 * @summary Gets the specified network profile in a specified resource group.
 * x-ms-original-file: specification/network/resource-manager/Microsoft.Network/stable/2024-03-01/examples/NetworkProfileGetConfigOnly.json
 */
async function getNetworkProfile() {
  const subscriptionId = process.env["NETWORK_SUBSCRIPTION_ID"] || "subid";
  const resourceGroupName = process.env["NETWORK_RESOURCE_GROUP"] || "rg1";
  const networkProfileName = "networkProfile1";
  const credential = new DefaultAzureCredential();
  const client = new NetworkManagementClient(credential, subscriptionId);
  const result = await client.networkProfiles.get(resourceGroupName, networkProfileName);
  console.log(result);
}
